src/com/telesign/util/TeleSignRequest.java

```diff
@@ -74,7 +74,7 @@
 	private boolean ts_date = false;
 
 	/** An AuthMethod enumeration value  that specifies the strength of the encryption method to use when signing the Authentication header. */
-	private AuthMethod auth = AuthMethod.SHA1;
+	private AuthMethod auth = AuthMethod.SHA256;

```

Comment:
This is only used for the strength of the encryption method to use when signing the Authentication header. This is different from the TLS cipher strength used for connections. Although I will hold on to this commit since changing the default of the encryption for Auth header is something we want to do.